src/jquery.ime.inputmethods.js

```diff
@@ -850,6 +850,26 @@
 			name: 'Pontic tilde',
 			source: 'rules/pnt/pnt-tilde.js'
 		},
+		'raj-bolnagri': {
```

Comment:
All these are not needed. Only the change later in the file, with `raj:`, is necessary.